### ISSUE TITLE
[Kubuntu 16.04][Dark-Breeze for gtk Windows theme] - enabling alternative column view makes column headers unreadable - white text on white background

### ISSUE THREAD

**Issue by JPRuehmann** [Aris-t2/CustomizeMyBird#5]
Hello if I enable alternative column view.
The column headers get displayed white on white with black bars.
how should it look like?

**Comment by Aris-t2** [comment]
Post a screenshot so I see what you mean.

I bet your non-default OS theme messes around with some of CMBs color settings.

**Comment by Aris-t2** [comment]
The "test 2" version might with this bug too.
https://github.com/Aris-t2/CustomizeMyBird/releases/download/2.0.7/CMB_v2.0.7.2.test.xpi

**Comment by JPRuehmann** [comment]
![screenshot_20180427_204251](https://user-images.githubusercontent.com/4321829/39379495-b68f36b2-4a5b-11e8-8d6a-cbc3fabfd319.png)

**Comment by Aris-t2** [comment]
Just to be sure, does this happens with "CMB_v2.0.7.2.test.xpi" too? I tried to force black font color for the affected option. Try to reinstall the file from that link again. I changed it shortly after first upload, because it missed something.

Sadly "Dark-Breeze for gtk Windows" does not appear on the available OS theme list as a downloadable theme, but the default "Dark Breeze" theme is present without causing those glitches. Not sure, if there is a proper solution for every case. This would mean the feature "Spaltenbeschreibung: alternative Darstellung" is not suitable for dark themes. The grey and blue colors it offers don't look good with dark themes anyway.
